test/dhtest.c

```diff
@@ -249,16 +249,18 @@ static int dh_computekey_range_test(void)
         || !TEST_true(DH_set0_pqg(dh, p, q, g)))
         goto err;
     p = q = g = NULL;
-    sz = DH_size(dh);
 
-    if (!TEST_ptr(buf = OPENSSL_malloc(sz))
+    if (!TEST_int_ge(sz = DH_size(dh), 0)
```

Comment:
Should it be TEST_int_gt? Continuing when sz == 0 does not make sense to me.

Reply:
Makes sense.  Fix pushed.